shared/events.py

```diff
@@ -1,13 +1,18 @@
+import logging
```

Comment:
Why import the old logger?
at least have failover or fallbacks

Reply:
> Why import the old logger? at least have failover or fallbacks

Will update so that it uses what was made in config_improvement